.gitignore

```diff
@@ -0,0 +1,40 @@
+build
```

Comment:
I would prefer not to add this very user-specific .gitignore file to the repo. Can you remove it?